.github/workflows/test.yaml

```diff
@@ -69,23 +69,23 @@ jobs:
       - name: Run UI tests (on template)
         run: sh ./scripts/tests-ui.sh
```

Comment:
I wonder if when we spin up `fuel-core` here, it's interacting with the other `fuel-core` from the published step?

Reply:
May be worth disabling this and seeing if it passes

Reply:
It's a nice idea, will try